src/cluster.c

```diff
@@ -2798,7 +2879,12 @@ int clusterProcessPacket(clusterLink *link) {
                     if (sender->slaveof)
                         clusterNodeRemoveSlave(sender->slaveof,sender);
                     clusterNodeAddSlave(master,sender);
+                    serverLog(LL_WARNING,
+                              "%.40s is now a replica of %.40s",
+                              sender->name,
+                              master->name);
```

Comment:
When I added these new trace statements, I was thinking that all topology change events are important and worth extra attention because they are not expected on a steady cluster (modulo rolling upgrades and slot migrations, of course). Now mulling over this some more, I think what I really wanted is a way to easily filter out all topology change events; whether or not these events are `WARNING` or `NOTICE` is actually less important to me. Would it work if I use a prefix for all these messages, say `TOPOLOGY:`?

Reply:
Adding topology is fine. We probably should add some standard about labeling of log messages anyways.

Reply:
I think per component logging is a good idea. Probably out of scope for this PR, just so we don't forget it I created the issue: https://github.com/redis/redis/issues/11825.